interpreter/README.md

```diff
@@ -22,10 +22,7 @@ Once you have OCaml, simply do
 ```
 make
 ```
-You'll get an executable named `./wasm`. This is a byte code executable. If you want a (faster) native code executable, do
-```
-make opt
-```
+You'll get an executable named `_build/default/wasm.exe`.
```

Comment:
Let's not break people's workflow. Can you tweak the makefile such that this is symlinked to ./wasm?

Reply:
I'm fine with having `./wasm` but IIRC, currently it's `./wasm.exe` and we get an error from dune saying that we're asking to generate `./wasm.exe` which is already in the source tree.